test/dot.pkg/dot.pkg.go

```diff
@@ -50,6 +50,11 @@ func PkgFuncs() error {
 	return errutil.Wrap(err)
 }
 
+func WithStack() error {
```

Comment:
Could we add another that does New(),With() and then an WithStack? Our existing chain should still be in there.

Reply:
👍 got it